.cirrus.yml

```diff
@@ -1,27 +1,38 @@
 gcp_credentials: ENCRYPTED[987a78af29b91ce8489594c9ab3fec21845bbe5ba68294b8f6def3cf0d380830f06687a89ea69c87344c5ade369700fe]
 
-gce_instance:
-  image_project: flutter-cirrus
-  image_name: flutter-engine-ubuntu1604
+gke_container:
+  image: gcr.io/flutter-cirrus/build-engine-image:0.1
+  cluster_name: build-16-cluster
   zone: us-central1-a
-  cpu: 24
-  memory: 32Gb
-  disk: 60
+  namespace: default
+  cpu: 16
```

Comment:
if your `build-16-cluster` cluster consists of 16 cores VMs then you can't use all 16 cores. There are some system pods are running. You probably want to change it to 14.